<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceleratedSpeedTestPage</class>
 <widget class="QWidget" name="AcceleratedSpeedTestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:1px solid white;
}</string>
  </property>
  <widget class="QLabel" name="lblTitle1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>终止速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>加减速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>110</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>距离</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTractionEndSpeed">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAcceleratePlus">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTractionDistance">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>280</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: yellow;
border: 0px;
color: black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>操作
提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTips">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>883</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string> 在司机室占有的条件下，单击按钮进行测试。
按钮按下表示激活测试，按键弹起表示取消测试。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>520</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255); 
	border:2px solid white; 
	border-left-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, 
                        stop: 0 gray, stop: 1 black); 			border-top-color: qlineargradient(x1: 0, y1: 0, x2: 0, 										y2: 1, stop: 0 gray, stop: 1 black);
  border-right-color: white; border-bottom-color: white;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>起始速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>工况</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>牵引</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTractionStartSpeed">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeStartSpeed">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeEndSpeed">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAccelerateMinus">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeDistance">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>210</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0m</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lblTitle1</zorder>
  <zorder>lblTitle1_2</zorder>
  <zorder>lblTitle1_3</zorder>
  <zorder>lblTractionEndSpeed</zorder>
  <zorder>lblAcceleratePlus</zorder>
  <zorder>lblTractionDistance</zorder>
  <zorder>btnTest</zorder>
  <zorder>lblTips</zorder>
  <zorder>btnBack</zorder>
  <zorder>lblTitle1_4</zorder>
  <zorder>lblTitle1_5</zorder>
  <zorder>lblTitle1_6</zorder>
  <zorder>lblTitle1_7</zorder>
  <zorder>lblTractionStartSpeed</zorder>
  <zorder>lblBrakeStartSpeed</zorder>
  <zorder>lblBrakeEndSpeed</zorder>
  <zorder>lblAccelerateMinus</zorder>
  <zorder>lblBrakeDistance</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
